### PR TITLE
fix bison build issue

### PR DESCRIPTION
**Pull request by thomasdstewart**
A few minor fixes that allow ike to build again on current buster.

### CODE REVIEW

File: debian/patches/bison-3-port.patch [hunk 1]
```diff
@@ -6,16 +6,23 @@ Subject: bison-3-port
...
-+++ b/source/iked/conf.parse.yy
+--- ike.orig/source/iked/conf.parse.yy
++++ ike/source/iked/conf.parse.yy
 @@ -41,7 +41,7 @@
  
  %skeleton "lalr1.cc"
  %defines
 -%define "parser_class_name" "conf_parser"
-+%define "parser_class_name" {conf_parser}
++%define api.parser.class {conf_parser}
```
Comment (yrro) [inline debian/patches/bison-3-port.patch:16]: I've applied this to master thanks

File: debian/patches/bison-3-port.patch [hunk 2]
```diff
@@ -6,16 +6,23 @@ Subject: bison-3-port
...
  
  %{
  
+@@ -56,7 +56,7 @@
+ %lex-param   { IKED & iked }
+ %locations
+ %debug
+-%error-verbose
++%define parse.error verbose
```
Comment (yrro) [inline debian/patches/bison-3-port.patch:25]: I the old directive still works so I'd prefer to leave this alone

File: debian/patches/bison-3-port.patch (hunk 1 above)
Comment (yrro) [inline debian/patches/bison-3-port.patch:16]: I've applied this to master thanks

File: debian/patches/bison-3-port.patch [hunk 3]
```diff
@@ -6,16 +6,23 @@ Subject: bison-3-port
...
-+%define "parser_class_name" {conf_parser}
++%define api.parser.class {conf_parser}
  
  %{
  
+@@ -56,7 +56,7 @@
+ %lex-param   { IKED & iked }
+ %locations
+ %debug
+-%error-verbose
```
Comment (yrro) [inline debian/patches/bison-3-port.patch:24]: ike builds without this so I'll leave it out if it doesn't change anything else

**Review by yrro — CHANGES_REQUESTED** [review]

File: debian/patches/bison-3-port.patch (hunk 1 above)
Comment (thomasdstewart) [reply debian/patches/bison-3-port.patch:16]: cool :)

File: debian/patches/bison-3-port.patch (hunk 3 above)
Comment (thomasdstewart) [reply debian/patches/bison-3-port.patch:24]: no worries :)

**Pull request closed without merging**